packages/tradershub/src/components/GetADerivAccountBanner/GetADerivAccountDialog.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
```

Comment:
Better to change the name of the folder to `GetDerivAccount`

Reply:
was just following the naming of existing folder / files
<img width="410" alt="image" src="https://github.com/binary-com/deriv-app/assets/108507236/12dde1b1-4bbb-40c8-8c8e-18f561e5854e">